packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js

```diff
@@ -68,6 +68,16 @@ export default class EmailProDomainDkimAutoconfigCtrl extends DkimAutoConfigurat
       .finally(() => {
         this.loading.step1 = false;
       });
+
+    if (this.dkimStatus === this.GLOBAL_DKIM_STATUS.NOK) {
+      this.services.EmailProDomains.getDomain(
```

Comment:
Is the api get in the service must be change, do not forget to remove the first parameter in the signature here

Reply:
done